src/main/kotlin/venus/glue/Renderer.kt

```diff
@@ -389,4 +401,51 @@ internal object Renderer {
 
         return "0x" + suffix
     }
+
+    private fun toUnsigned(value: Int): String {
+        var remainder = value.toLong()
+        var suffix = ""
+
+        // output as two's complement
+        if (remainder < 0) {
+            remainder += 0x1_0000_0000L
+        }
+
+        // convert to hex
+        while (remainder > 0) {
+            val hexDigit = hexMap[(remainder % 10).toInt()]
+            suffix = hexDigit + suffix
+            remainder /= 10
+        }
+
+        if (suffix.length < 1) {
+            suffix = "0"
+        }
+        return suffix
+    }
+
+    /**
+     * Sets the display type for all of the registers and memory
+     * Rerenders after
+     */
+    fun setDisplay(dis_type: String) {
+        display_type = dis_type
+        optSetVisibily(dis_type)
+        updateAll()
+    }
+
+    private val opts = listOf("hex", "dec", "unsign")
+
+    private fun optSetVisibily(opt: String) {
+        var tabDisplay: HTMLElement?
+        for (option in opts) {
+            if (opt.equals(option)) {
+                tabDisplay = document.getElementById("$option-opt") as HTMLElement
+                tabDisplay.className = "is-active"
```

Comment:
We could make this a single line using the `if` statement:

    tabDisplay.className = if (opt == option) "is-active" else ""